src/util/paths.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+const repoRoot = process.cwd(); // expect CLI to be run from consumer repo root
+const appPath = path.resolve(repoRoot, 'src');
+
+module.exports = {
+	repoRoot,
+	appPath,
+};
```

Comment:
Okay. I'll file a ticket to review the organization of the various `config` modules in this repo now that we have a better sense of where they need to be consumed. In the meantime, I think it would be better to define `repoRoot` and `appPath` once - either in this script or `src/commands/buildCommands/config/paths`, but not both.